.github/workflows/updateVersionsAndCreatePR.yml

```diff
@@ -0,0 +1,85 @@
+        run: |
+          sudo apt-get update -y
+          sudo apt install gh
+
+      - name: Create Branch
+        run: |
+          version_name="${{ github.event.inputs.version }}"
+          branch_name="update-versions-${version_name}"
+          git checkout -b "$branch_name"
+
+      - name: Get and Change Component Versions
+        id: get_versions
+        run: |
+          components=("perf-timing" "imperative" "cli-test-utils" "secrets-for-zowe-sdk" "core-for-zowe-sdk" "zos-uss-for-zowe-sdk" "provisioning-for-zowe-sdk" "zos-console-for-zowe-sdk" "zos-files-for-zowe-sdk" "zos-logs-for-zowe-sdk" "zosmf-for-zowe-sdk" "zos-workflows-for-zowe-sdk" "zos-jobs-for-zowe-sdk" "zos-tso-for-zowe-sdk" "cli" "cics-for-zowe-cli" "db2-for-zowe-cli" "ims-for-zowe-cli" "mq-for-zowe-cli" "zos-ftp-for-zowe-cli")
+          zowe_version="${{ github.event.inputs.zowe_version }}"
+          version="${{ github.event.inputs.version }}"
+          if [[ "$zowe_version" == "zowe-v1-lts" ]]; then
+            components+=("secure-credential-store-for-zowe-cli")
+          fi
+          # echoing to txt file to later add this file as the PR description
+          echo "This PR updates the following package versions in zowe-versions.yaml for \`$zowe_version\`" >> description.txt
+          echo "| Updated Package | Old Version | New Version |" >> description.txt
+          echo "| --- | --- | --- |" >> description.txt
+          for component in "${components[@]}"; do
+            current_version=$(grep -A 2 "^  $component:" zowe-versions.yaml | grep "$zowe_version" | awk '{print $2}')
+            new_version=$(npm view @zowe/$component "dist-tags.$zowe_version")
+            if [[ "$new_version" != "$current_version" ]]; then
+              echo "|$component | $current_version | $new_version|" >> description.txt
+              # Replaces the version number. If it doesn't match on the first line after the component, it goes to the next line and checks again.
+              sed -i -E "/^  $component:/ { n; s/($zowe_version: ).*/\1$new_version/; t; n; s/($zowe_version: ).*/\1$new_version/ }" zowe-versions.yaml
+            fi
+          done
+
+          # Update zowe-versions.yaml tags section
+          sed -i -E "/$zowe_version:/ { n; s/(version: ).*/\1$version/ }" zowe-versions.yaml
+
+
+      - name: Commit and Push Changes
+        run: |
+          git config --global user.name "GitHub Actions"
+          git config --global user.email "actions@github.com"
+          git commit -am "Update component versions"
+          git push origin HEAD
+
+      - name: Create Pull Request
+        run: |
+          version_name="${{ github.event.inputs.version }}"
+          branch_name="update-versions-${version_name}"
+          pr_title="Update zowe component versions for $version_name"
+          body_file=description.txt
+          gh pr create --base master --head "$branch_name" --title "$pr_title" --body-file "$body_file" --reviewer atorrise

```

Comment:
Is it possible to add multiple reviewers to the PR when creating it with GitHub CLI? It would be nice to automatically add everyone that's usually involved w/ the Zowe release PRs, in case some of us are OOO/caught up with other work 🙂

Reply:
@traeok  yes it's possible: 
>gh pr create --base master --head "$branch_name" --title "$pr_title" --body-file "$body_file"
>gh pr review-request --reviewer atorrise
>gh pr review-request --reviewer another_reviewer

but don't we pretty much just add OJ as the reviewer? I was going off our internal documentation

Reply:
For `cli-standalone-package`, the team usually reviews those PRs - but for `zowe-install-packaging`, I think just adding OJ will be fine 👍

Reply:
@traeok oop ok 😅 do you think i should just add the whole team? I'm probably going to test that this works right now so you might get a notification from one of my personal repos

Reply:
That sounds good to me 😁

Reply:
Okay i'm not sure if my fix works exactly because in my test repo, none of you guys are contributors so im getting error messages with `gh pr edit --add-reviewer`  because "..specified GitHub username doesn't exist in the repository or organization [I am] working with"

will have to test once merged. Also this means that adding Gene and Kevin will be an issue since they haven't contributed to this repo yet so I'll probably leave them out